### ISSUE TITLE
Interface adapted to mobile screens

### ISSUE THREAD

**Issue by alin256** [CodingTrain/Directional-Boring#26]
@DenisovichDev can we update the interface for the phone screen.

I am thinking to have a large **toggle bias** button in the middle and all the other controls closer to the bottom:

![interface-concept](https://user-images.githubusercontent.com/7563037/177975503-1b492c0d-c2e1-426e-ba5d-d4fe374c4e88.png)

@holsaeter

**Comment by DenisovichDev** [comment]
Absolutely! It's a great concept, I'll start working on it. Might have change some bits on `sketch.js` to adapt on mobile devices

**Comment by alin256** [comment]
I tried it on iPad/iPhone yesterday, and the button interface is laggy. I suppose it is an issue with WebKit touch implementation.

Did anyone try on Android?

I suggest having some kind of direct touch controls on the main canvas for steering. Those seem to have much less lag in my testing.

**Comment by shiffman** [comment]
Yes, I would agree! Should we detect mobile and serve a different view or create a version that works at any resolution / aspect ratio?

**Comment by alin256** [comment]
Since it is a single-view application it is easier to just **rescale it according to the screen and remove the scrolls**.

We did a while ago for http://game.geosteering.no/geo/redirect
The code is here: https://github.com/NORCE-Energy/geosteering-game-gui 

P5's support for scaling is quite good. 
So it is a matter of preventing the scrolling in the CSS.
I can try something soon.

**Comment by alin256** [comment]
I feel like with keyboard and click-on-interface controls we can and should get rid of all buttons except for **pull back**

Demo here: https://al-digital.no/Directional-Boring/

![image](https://user-images.githubusercontent.com/7563037/178152130-8d6356d0-15d1-4b56-9229-cfa96b847e1b.png)

**Comment by alin256** [comment]
@DenisovichDev the improved CSS did not scale correctly on iPhone. Do you know how to fix? 
![Uploading A032FF69-CF6D-4AA5-A5CB-440656DA1DF0.png…]()

**Comment by alin256** [comment]
![2821AAC8-099C-4BA6-B4F7-5A56E0B50E0A](https://user-images.githubusercontent.com/7563037/179040305-c0d663e9-76e6-45ed-ba3f-8195b4491a7c.png)

**Comment by DenisovichDev** [comment]
That is really weird... It's working fine in Android. I'll look into it

**Comment by alin256** [comment]
> It's working fine in Android. I'll look into it

Can you check that the updated version did not become worse on Android?

**Comment by DenisovichDev** [comment]
It's not something Android specific, but the style sheets need work. I'll get into it tonight

**Comment by alin256** [comment]
For completeness, the solution to the iOS problems was in a magic line in the HTML file:

```
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

and a magic line in the CSS file
```
  -webkit-text-size-adjust: none;
```
